datadog/cassettes/TestAccDatadogIntegrationAWS.yaml

```diff
@@ -2,17 +2,22 @@
 version: 1
 interactions:
 - request:
-    body: '{"account_id":"1234567888","role_name":"testacc-datadog-integration-role","filter_tags":[],"host_tags":[],"account_specific_namespace_rules":{}}'
+    body: |
+      {"account_id":"1234567888","account_specific_namespace_rules":{},"role_name":"testacc-datadog-integration-role"}
     form: {}
     headers:
+      Accept:
+      - application/json
       Content-Type:
       - application/json
+      Dd-Operation-Id:
+      - CreateAWSAccount
       User-Agent:
-      - Datadog/dev/terraform (go1.13.4)
+      - datadog-api-client-go/1.0.0-beta.1+dev.1 (go go1.13; os darwin; arch amd64)
```

Comment:
we need to update the `User-Agent` for `datadog-api-client-go` to `terraform-provider-datadog/{version} datadog-api-client-go/1.0.0-beta.1+dev.1 (go go1.13; os darwin; arch amd64)` format.

Reply:
I will do that in another PR because it will be too many changes for this PR. Good catch and let's update it once we have `beta.3`

Reply:
Could we instead ignore the user agent in the cassettes ? It's really system and version dependent, so not great for replaying as people won't always be running tests on the same system

Reply:
What I meant is that we are recording new cassette but the user-agent is not right.